### DISCUSSION TITLE
Trying to connect to a camera live stream

### DISCUSSION THREAD

**Discussion by tomer2141** [bluenviron/mediamtx#781] [Questions]
I'm trying to connect my camera to the rtsp-simple-server. but when the server wants to open a connection with the output stream I'm getting this error:
closed (H264 track 1 is not valid: sprop-parameter-sets is missing (96 packetization-mode=1))

I'm pretty new to this so I hope my explanation of the problem is good enough :)

**Comment by aler9** [comment]
Hello, this issue has been fixed since some v0.17.10 (see #411) - please use the latest version.

**Reply by KaleiRo** [reply]
Hi,

I have the same issue with latest version on RPI4, the problem exist when you use h264 hardware encoding with ffmpeg. when you use software encoding it's ok.

[a comment by an automated account is not shown]
